Interop/Runtime/src/native/kotlin/kotlinx/cinterop/NativeTypes.kt

```diff
@@ -56,6 +56,20 @@ external fun CPointer<*>.getRawValue(): NativePtr
 
 internal fun CPointer<*>.cPointerToString() = "CPointer(raw=$rawValue)"
 
+
+public final class NativeVector private constructor()
+
+public class VectorVarOf<T : NativeVector>(rawPtr: NativePtr) : CVariable(rawPtr)
+
+public typealias VectorVar = VectorVarOf<NativeVector>
```

Comment:
Should also be included here:
https://github.com/JetBrains/kotlin-native/blob/e1b3fefd3333fa730bb00174b7eb24c90822f6a0/Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrType.kt#L90
cc @sbogolepov

Reply:
Correct. Please note, that the name of `typealias` should changed be `NativeVectorVar` because this code relies on such naming convention.